From f400ab0c9aac70794867053e9a5b64361c33f310 Mon Sep 17 00:00:00 2001
From: Brice Dobry <brice.dobry@futurewei.com>
Date: Tue, 7 Sep 2021 06:23:39 -0700
Subject: [PATCH] Add Lu Yahan to RISC-V owners list

Lu Yahan is a key contributor to the RISC-V target and I would like to
add him to the owners list so that he can approve changes in Gerrit.

Change-Id: I017fb2ef20320887959e9830fb63e05df121c7b9
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3145370
Reviewed-by: Ji Qiu <qiuji@iscas.ac.cn>
Reviewed-by: Hannes Payer <hpayer@chromium.org>
Commit-Queue: Brice Dobry <brice.dobry@futurewei.com>
Cr-Commit-Position: refs/heads/main@{#76800}
---
 RISCV_OWNERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/RISCV_OWNERS b/RISCV_OWNERS
index 8f8e15a40a0..e3e11fdf494 100644
--- a/RISCV_OWNERS
+++ b/RISCV_OWNERS
@@ -1,3 +1,4 @@
 brice.dobry@futurewei.com
 peng.w@rioslab.org
 qiuji@iscas.ac.cn
+yahan@iscas.ac.cn
-- 
2.35.1

